### PR TITLE
feat(FreeWater): add presence

### PR DESCRIPTION
**Pull request by CodesWithJames**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots

image
image
image

### CODE REVIEW

**Comment by DarkVillager** [comment]
> * [✓] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
> * [✓] I linted the code by running `yarn format`
> * [✓] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
Just change the checkmars to the letter X

**Comment by CodesWithJames** [comment]
Thank you

File: websites/F/FreeWater/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	  "name": "Jamesiboi",
+	  "id": "785575171599499295"
+	},
+	"service": "FreeWater",
+	"description": {
+	  "en": "FreeWater is an advertising platform that utilizes premium spring water as a new type of advertising medium.",
+	  "sv": "FreeWater är en markandsförings plattform som använder premium källvatten som en ny marknadsföringsmedium."
+	},
+	"url": "www.freewater.io",
+	"version": "0.0.1",
```
Comment (theusaf) [inline websites/F/FreeWater/dist/metadata.json:13]:
```suggestion
	"version": "1.0.0",
```

**Review by theusaf — CHANGES_REQUESTED** [review]
Also, resolve CI.

File: websites/F/FreeWater/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+	  "name": "Jamesiboi",
+	  "id": "785575171599499295"
+	},
+	"service": "FreeWater",
+	"description": {
+	  "en": "FreeWater is an advertising platform that utilizes premium spring water as a new type of advertising medium.",
+	  "sv": "FreeWater är en markandsförings plattform som använder premium källvatten som en ny marknadsföringsmedium."
```
[the remark this answers is not in the document]
Comment (CodesWithJames) [reply websites/F/FreeWater/dist/metadata.json:10]: I checked the docs, and they are.

File: websites/F/FreeWater/presence.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+    clientId: "1019263816070082582",
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo",
+        smallImageKey: "reading",
```
[the remark this answers is not in the document]
Comment (CodesWithJames) [reply websites/F/FreeWater/presence.ts:9]: Okay

**Comment by RisingSunLight42** [comment]
Fix errors : 
Images links are supposed to be imgur links. Plus, a language given doesn't exist, please check those errors.

**Comment by EncryptedDev** [comment]
Low quality presence

**Pull request closed without merging**